config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  root to: "homepages#index"
+
+  resources :trips
+  resources :passengers do
+    resources :trips, only: [:index, :new]
+  end
+  resources :drivers do
+    resources :trips, only: [:index]
+  end
+  resources :homepages
```

Comment:
y'all probably didn't need all of `resources :homepages` since there's truly one route that you care about; it'll be best to be selective here!